### PR TITLE
UUPS proxy e2e and fixes

### PR DESCRIPTION
**Pull request by ioedeveloper**
E2E Tests for https://github.com/ethereum/remix-project/pull/2260 and https://github.com/ethereum/remix-project/pull/2594

### CODE REVIEW

File: apps/remix-ide-e2e/src/tests/proxy.test.ts [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+      .waitForElementVisible('[data-id="confirmProxyDeployment-modal-footer-ok-react"]')
+      .click('[data-id="confirmProxyDeployment-modal-footer-ok-react"]')
+      .waitForElementPresent('[data-id="universalDappUiTitleExpander0"]')
+      .waitForElementPresent('[data-id="universalDappUiTitleExpander1"]')
+  },
+
+  'Should interact with deployed contract via ERC1967 (proxy)': function (browser: NightwatchBrowser) {
+    browser
+      .getAddressAtPosition(1, (address) => {
+        console.log('Proxy address: ' + address)
```
Comment (Aniket-Engg) [inline apps/remix-ide-e2e/src/tests/proxy.test.ts:70]: log should be removed

File: apps/remix-ide-e2e/src/tests/proxy.test.ts [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+      .waitForElementVisible('[data-id="confirmProxyDeployment-modal-footer-ok-react"]')
+      .click('[data-id="confirmProxyDeployment-modal-footer-ok-react"]')
+      .waitForElementPresent('[data-id="universalDappUiTitleExpander0"]')
+      .waitForElementPresent('[data-id="universalDappUiTitleExpander1"]')
+  },
+
+  'Should interact with initialized contract to verify parameters': function (browser: NightwatchBrowser) {
+    browser
+      .getAddressAtPosition(1, (address) => {
+        console.log('Proxy address: ' + address)
```
Comment (Aniket-Engg) [inline apps/remix-ide-e2e/src/tests/proxy.test.ts:118]: same here

File: apps/remix-ide-e2e/src/tests/proxy.test.ts (hunk 1 above)
Comment (ioedeveloper) [reply apps/remix-ide-e2e/src/tests/proxy.test.ts:70]: Console logs should be left for easy debugging in circleci.

File: apps/remix-ide-e2e/src/tests/proxy.test.ts (hunk 2 above)
Comment (ioedeveloper) [reply apps/remix-ide-e2e/src/tests/proxy.test.ts:118]: Console logs should be left for easy debugging in circleci.

**Review by Aniket-Engg — APPROVED** [review]

**Pull request merged**